neo/SmartContract/ContractParameter.py

```diff
@@ -55,6 +53,12 @@ def ToParameter(item: StackItem):
             return ContractParameter(type=ContractParameterType.InteropInterface, value=item.GetInterface())
 
     def ToJson(self):
+        """
+        Converts a ContractParameter instance to a json representation
+
+        Returns:
+            json object
```

Comment:
The convention I used in the past was:
```
        """
        Convert object members to a dictionary that can be parsed as JSON.

        Returns:
             dict:
        """
```
The description isn't that important, but I believe the `return` type could be more explicit (as there's no json object type in python afaik)